cmd/util.go

```diff
@@ -2,6 +2,10 @@ package cmd
 
 import "github.com/pkg/errors"
 
+var (
+	Namespace string
```

Comment:
A comment explaining that this is used by subcommands might be nice.

It starting to get a bit chaotic. We have variables for flags in `root.go` and in every subcommand. Flag handling will probably require some refactoring really soon.